desktop/autoupdate/autoupdate.go

```diff
@@ -28,18 +24,13 @@ var (
 
 	cfgMutex           sync.RWMutex
 	watchForUpdateOnce sync.Once
-	httpClient         atomic.Value
 	fnIconURL          func() string
 )
 
 // Configure sets the CA certificate to pin for the TLS auto-update connection.
 func Configure(updateURL, updateCA string, iconURL func() string) {
 	setUpdateURL(updateURL)
 	fnIconURL = iconURL
-	httpClient.Store(
-		&http.Client{
-			Transport: proxied.ChainedThenFrontedWith(updateCA),
-		})
```

Comment:
We are losing track of the update CA here -- have to think on that one

Reply:
hmnn...actually I think this will break autoupdates as it stands

Reply:
Ah interesting -- the autoupdate stuff actually handles this itself:

https://github.com/getlantern/autoupdate/blob/master/autoupdate.go#L122